src/node_file.cc

```diff
@@ -3366,6 +3418,7 @@ static void CreatePerIsolateProperties(IsolateData* isolate_data,
   SetMethod(isolate, target, "ftruncate", FTruncate);
   SetMethod(isolate, target, "rmdir", RMDir);
   SetMethod(isolate, target, "mkdir", MKDir);
+  SetMethodNoSideEffect(isolate, target, "mkdirSync", MKDirSync);
```

Comment:
This needs to be SetMethod since filesystem operations should not be pre executed on repl before pressing enter.